awsconfigrules/build.f/particles-awsconfigrules/particles/cftemplates/template.json.hbs

```diff
@@ -19,4 +20,5 @@
   {{output "fetch_rules_lambda_arn" logicalId="FetchRulesLambdaArn"}}
   {{output "sg_portrange_lambda_arn" logicalId="SGPortRangeLambdaArn"}}
   {{output "ec2_create_trigger_db_update_lambda_arn" logicalId="EC2CreateTriggerDbUpdateLambdaArn"}}
+  {{output "ec2_create_trigger_db_update_lambda_proxy_arn" logicalId="EC2CreateTriggerDbUpdateLambdaProxyArn"}}
```

Comment:
Rename output to ec2_create_trigger_master_lambda_proxy_arn
Rename logicalId to EC2CreateTriggerMasterLambdaProxyArn